content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -46,10 +49,10 @@ for details.
 apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
-  name: standard
+  name: sc-name
 provisioner: kubernetes.io/aws-ebs
 parameters:
-  type: gp2
+  pools: pool1,pool2,pool3
```

Comment:
Is this a parameter that the `kubernetes.io/aws-ebs` storage driver understands?

Reply:
i think the issue needed to be adding the pool should i raise a PR with documentation changes without yaml file